.envrc

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+use_trunk.io() {
+  if [[ -f .trunk/tools/trunk ]]; then
+    trunk=.trunk/tools/trunk
+  else
+    tmp_file=$(mktemp)
+    trap 'rm -f ${tmp_file}' EXIT
+    curl -fsSL https://trunk.io/releases/trunk -o "${tmp_file}"
+    chmod +x "${tmp_file}"
+    trunk="${tmp_file}"
+  fi
+
+  "${trunk}" daemon start
+}
+
+use trunk.io
```

Comment:
Just want to verify that this isn't a typo.

Reply:
Seems like this is intended:
https://github.com/direnv/direnv/blob/9c53eaeab6bff27123d51b81f4831d9f454d548b/stdlib.sh#L1083-L1101